### DISCUSSION TITLE
Copilot Chat is enabled on my account, but I'm not seeing it on my VSCode

### DISCUSSION THREAD

**Discussion by swrh** [community/community#56121] [Copilot Conversations] [answered]
### Copilot Chat

Copilot Chat is enabled on my account. Why am I not able to see or use it on VSCode **INSIDERS**?

I received the email saying that Copilot Chat is now enabled on my account, I followed the instructions to enable it on my VSCode, but I still don't see it on my activity bar.

Do I have to do something else to make it work?

**Comment by Akash1134** [comment]
Hi @swrh ,

To access Copilot Chat you Copilot Nightly extension ([Download it here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly)). If you already have it, please ensure that you have the following:
- An active GitHub subscription ( ❗  Please ensure the GitHub ID you intend to use has an active subscription to GitHub Copilot and matches the ID that you used to join the GitHub Copilot chat waitlist)
- The Visual Studio Code - [Insiders build](https://code.visualstudio.com/insiders/)

Thanks!

**Reply by swrh** [reply]
I had already done all of that (installed and set up Visual Studio Code Insiders and GitHub Copilot Nightly, logged off and on GitHub), as described on the initial installation, but it didn't work.

**Comment by STEFERD7** [comment]
In the Visual Studio Code Marketplace, go to the GitHub Copilot extension page and click Install. A popup will appear, asking to open Visual Studio Code. Click Open Visual Studio Code. In the "Extension: GitHub Copilot" tab in Visual Studio Code, click Install.
Regards:http://cpsandtypingtest.com/2-minute-typing

**Reply by swrh** [reply]
I had already done all of that (installed and set up Visual Studio Code Insiders and GitHub Copilot Nightly, logged off and on GitHub), as described on the initial installation, but it didn't work.

**Comment by STEFERD7** [comment]
In the Visual Studio Code Marketplace, go to the GitHub Copilot extension page and click Install. A popup will appear, asking to open Visual Studio Code. Click Open Visual Studio Code. In the "Extension: GitHub Copilot" tab in Visual Studio Code, click Install.
regards: [cpsandtypingtest](https://github.com/orgs/community/discussions/56121)

**Comment by Lbyn96** [comment]
You need to install Visual Studio Code - Insiders and use Visual Studio Code - Insiders in order to see it.

**Reply by swrh** [reply]
I followed their instructions, which mentioned installing VSCode Insiders, and it didn't work.

**Comment by Armanasq** [comment]
Same here. I've got the email, followed the instructions but still cannot seeing the icon.
I am using Ubuntu 20.04

**Comment by somedaygone** [comment, marked as answer]
It wasn't working for me. I got it working finally. 

1. I thought I had installed VS Code Insider, but it wasn't. If it's installed, the VS Code icon should be green instead of blue. I don't know what happened here, but the first time I tried with the Windows "system" installer, and then I did the "user" install and that one showed up.

2. Then I had to install both the GitHub Copliot Nightly extension **and** the GitHub Copilot Chat extension. 
3. Then I had to click the Accounts button on the Nav bar and allow Copilot to signin to GitHub.

After this, a chat icon showed up at the bottom of my Nav bar. Pure bliss.

**Reply by swrh** [reply]
Thank you so much! That's what it was. After uninstalling the regular VSCode and installing VSCode Insiders I got the Copilot Chat working! Thank you!

**Comment by pchettri3** [comment]
I have tried on insider and vscode both. I am using nightly version with chat. I still do not see chat icon. I have $10 subscription per month

**Reply by honorsystem** [reply]
same

**Comment by gerge4567** [comment]
If you are disabling GitHub Copilot, you will be asked whether you want to disable suggestions globally, or for the language of the file you are currently editing. In the upper-right corner of any page, click your profile photo, then click Settings. In the "Code, planning, and automation" section of the sidebar, click GitHub Copilot. On the GitHub[(-)](uhttps://carparkingzoid.com/2023/10/30/download-the-old-version-of-car-parking-multiplayer-mod-apk/rl) Copilot settings page, click Enable GitHub Copilot .In the Visual Studio Code activity bar, click the GitHub Copilot Chat icon to open the GitHub Copilot Chat chat window. At the bottom of the GitHub Copilot Chat window, in the Ask Copilot a question or type / for topics text box, type a coding related question, then press Enter.In the upper-right corner of GitHub.com, select your profile photo, then click Your organizations. Next to the organization, click Settings. In the "Access" section of the sidebar, click Billing and plans. Under "Copilot", view the breakdown of your GitHub Copilot usage and upcoming changes in your organization.
